docs/source/contributing/version-policy.md

```diff
@@ -16,10 +16,16 @@ This chapter describes the versions of software that Volto supports and how Volt
 
 (volto-generator-compatibility-with-volto-label)=
 
-## `volto-generator` compatibility with Volto
+## `@plone/volto-generator` compatibility with Volto
 
-`volto-generator` is a tool that generates a Volto project.
-The following table describes the compatibility between versions of `volto-generator` and Volto.
+```{deprecated} Volto 18.0.0
+`@plone/volto-generator` was deprecated in Volto 18.0.0.
+It is not maintained, and you should not use it.
+For new projects, use [Cookieplone](https://github.com/plone/cookieplone) instead.
+```
+
+`@plone/volto-generator` is a tool that generates a Volto project.
+The following table describes the compatibility between versions of `@plone/volto-generator` and Volto.
 
 | Generator version | Volto version |
 |-------------------|---------------|
```

Comment:
The next line makes it sound like it's still supported for Volto 18, we need to change that.

Reply:
Does it work in 18? I assumed that if something works, then that means it's supported.

Also if something works, it can also be deprecated.

Reply:
No, we started to update it for Volto 18 but then realized we needed to create Cookieplone instead, because the structure was too different.

Reply:
Since it doesn't work in 18, then why do we even have it in 18? https://github.com/plone/volto/tree/main/packages/generator-volto

Reply:
"It does work" but it's true that maybe we should not advertise it, it could be confusing.

Reply:
Well, I just tried it, and [it didn't work for me](https://github.com/plone/training/pull/901#issuecomment-2547774654). But it could that the [Training I followed](https://training.plone.org/voltoaddons/01-addon-basics.html#bootstrap-a-new-volto-project) has incorrect information.

Reply:
@stevepiercy there's still an acceptance test for it in place. So it should still work for 18. For project, the add-on template is not being tested. What did you do? That training could be outdated.

Reply:
I followed the exact instructions published in that training, pretending to be a learner.

Reply:
> there's still an acceptance test for it in place. So it should still work for 18. For project, the add-on template is not being tested.

@sneridagh When I create an app with it on Volto 18, the tests don't work. I get the error:

```bash
● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/en/ecmascript-modules for how to enable it.
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /app/node_modules/@plone/registry/dist/index.js:339
    export {
    ^^^^^^

    SyntaxError: Unexpected token 'export'

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1350:14)
      at Object.<anonymous> (node_modules/@plone/volto/src/registry.js:8:40)
      at Object.<anonymous> (node_modules/@plone/volto/test-setup-config.jsx:6:40)
```
Do we want to fix this?

Reply:
By the way, how do I create an equivalent of a `@plone/volto-generator` app in cookieplone? It seems we only have templates for addons?

Reply:
The version of @plone/volto-generator for Volto 18 was created quite early in the process of developing Volto 18, and it hasn't really gotten much attention as further changes have happened after we switched to recommending cookieplone. It doesn't surprise me that it's not working fully, and I don't think it's worth a lot of effort to fix it. We need to update trainings to use cookieplone or be clear that they are for an older version of Volto.

@wesleybl There's a new structure for Volto 18, and there's no longer a distinction between an app and an addon. (Similar to how in the backend a project and an addon are both just Python packages.) Use the `project` template in cookieplone if you want to generate both a backend and a frontend, or the `frontend_addon` template if you only want to generate a frontend (for use with an existing backend).

Reply:
@sneridagh this came up in https://github.com/plone/volto/issues/6528#issuecomment-2551178777.

Reply:
If `@plone/volto-generator` doesn't work 100% with Volto 18, I also think it should be removed from the compatibility table.